Doc/whatsnew/3.7.rst

```diff
@@ -240,12 +322,12 @@ module:
 * :func:`time.process_time_ns`
 * :func:`time.time_ns`
 
-While similar to the existing functions without the ``_ns`` suffix, they
-provide nanosecond resolution: they return a number of nanoseconds as a Python
-``int``.
+The new functions are similar in function to the existing functions
+without the ``_ns`` suffix, but instead of returning fractional seconds,
```

Comment:
...suffix. They differ by returning nanoseconds as integers, instead of fractional seconds.